### PR TITLE
Fix homepage zooming on iPhone

### LINKED ISSUE #2151: Browser zoom-in the search input on mobile when typing
## Description
In staging environment, when tapping the search bar in homepage to start typing, the browser zoom-in the search area. This behavior interrupts the natural flow and becomes very frustrating.

## Reproduction
1. Go to the [staging's homepage](https://staging.openverse.org/) on a mobile device
2. Tap the search bar to type a search term
3. See how the browser zoom-in the search area

### Firefox on iOS 16.3
https://user-images.githubusercontent.com/895819/217201149-f1071bad-2559-4df2-938e-7e3612cfdcd9.MP4

## Environment
Staging

## Additional context
I remember we solved this in the previous header by setting the font size to `16px`. For that same reason, the [input's placeholder](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=3835%3A11069&t=Lr82Wad6bPU8zk0K-4) in the new header has two interaction variants: Click and Touch.

### Click variant
In this variant, the font size follows the `Label regular` style, to be consistent with the other UI components of the header.

CleanShot 2023-02-07 at 10 20 16@2x

### Touch variant
In this variant, the font size is `16px` to behave fluently on mobile devices.

CleanShot 2023-02-07 at 10 20 28@2x

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #2151 by @panchovm 

## Description

This PR sets the input text to use `paragraph-large` style (16px, line-height 190%) for widths less than `md`.

`paragraph-large` style was using `leading-larger`, which was causing not-found errors during build. I replaced it with `leading-[190%]`. The line-height tailwind styles need to be fixed and will be fixed in later PRs.

Also, WebStorm complains that `semibold` is incorrect, so I replaced it with `semi-bold` :)

## Testing Instructions

Use iPhone and search for something from the homepage. You should not see the zooming as in the description of the linked issue.
iPhoneZooming

## Checklist

- [x] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by panchovm — APPROVED** [review]
It looks excellent. Great work ✨

File: src/styles/tailwind.css [hunk 1]
```diff
@@ -43,19 +43,19 @@ Heading 1           - 90px - 7xl bold
 Heading 2           - 32px - 6xl bold
 Heading 3           - 28px - 5xl bold
 Heading 4           - 24px - 4xl bold
-Heading 5           - 19px - 3xl semibold
-Heading 6           - 16px - 2xl semibold
```
Comment (dhruvkb) [inline src/styles/tailwind.css:47]: Nit: 'semibold' is a valid word & the name of the corresponding Tailwind class.

**Review by dhruvkb — APPROVED** [review]
LGTM!

**Review by zackkrida — APPROVED** [review]
What a relief! LGTM.

**Pull request merged**